10-setup-htcondor.sh

```diff
@@ -235,11 +236,7 @@ echo
 cd $LOCAL_DIR
 
 # gwms files in the correct location
-cp -a /gwms/* .
-mkdir -p .gwms.d/bin
-for target in cleanup  postjob  prejob  setup  setup_singularity; do
-    mkdir -p .gwms.d/exec/$target
-done
+cp -a /gwms/. $LOCAL_DIR/
```

Comment:
I guess that's an interesting way to avoid putting things under `$LOCAL_DIR/gwms/`, right?

Looks like it'll do the trick.

Reply:
Well that and `cp -a /gwms/*` was not picking up the `.gwms*` dirs

Reply:
Ah, is that (picking up the `.gwms*` dirs) what was meant by "Ensure that we run per-job module setup" ?

Reply:
That's part of it, the other bit that enables this is `/gwms/.gwms.d/exec/prejob/setup_prejob.sh`, which was added in the same commit

Reply:
Ah right, with the curl command.  Very trusting of those glideinWMS folks ;)

Thanks for the explanation.